### DISCUSSION TITLE
New to Contributing, Getting Illegal character in jreinfo.dll on Launch

### DISCUSSION THREAD

**Discussion by SVNKoch** [eclipse-oomph/oomph#18] [General]
Hey there 
I want to contribute to the oomph project. But I'm struggeling to get the codebase working right now.
I used the eclipse installer and set up an Eclipse IDE for Eclipse Committers and choose the oomph project.
Now I wanted to launch the code using the "Setup (Installer Dialog)" or "Setup (Installer Dialog Online)" launch config, but I'm getting an error after startup that there is an Illegal character at index 33 in the jreinfo.dll file. I validated the plugin list and no errors were detected.

Here is the full stacktrace: [stacktrace.txt](https://github.com/eclipse-oomph/oomph/files/12361011/stacktrace.txt)

(I'm on Windows 10)

Best regards
Sven :)

_PS: maybe your [CONTRIBUTING.md](https://github.com/eclipse-oomph/oomph/blob/master/CONTRIBUTING.md) file could get improved. You are linking to the developer page on eclipse.org for resources which itself just refers you back to github. Furthermore you link to https://ci.eclipse.org/oomph/ for further instructions for setting up a development environment - this page is (currently) 503._

**Comment by merks** [comment]
Sorry, this stack trace is a recently introduced regression and it's really great that you reported it!  The problem is that the URL for accessing *.dll, as produced by the platform, unfortunately contains a space character and URL.toURI does not properly deal with this.

That's fixed now by this:

https://github.com/eclipse-oomph/oomph/commit/828b3c9acb75a98b4858d31b607b0032ab181169

You should be able to do a pull and the that particular problem will go away.

Yesterday the ci.eclipse.org instances were all broken for quite a few hours so that was quite unhelpful.  😱 

The CONTRIBUTING.md is indeed very crappy and really quite broken; it has not been updated since the migration from git.eclipse.org to github.com. 😱 I will try to find time ASAP to fix it.

Hopefully you aren't too discouraged!

**Reply by merks** [reply]
I added a readme:

https://github.com/eclipse-oomph/oomph#readme

and simplified the contribution page:

https://github.com/eclipse-oomph/oomph/blob/master/CONTRIBUTING.md

There is still room for further improvement.

**Reply by SVNKoch** [reply]
Thanks you very much for you quick response and also fixing the issue. I was able to now start the launch config succesfully.
And thanks for updating the CONTRIBUTING.md file, much appreciated!
